### ISSUE TITLE
Layout shift on news overview page

### ISSUE THREAD

**Issue by dmijatovic** [research-software-directory/RSD-as-a-service#1823]
When using ... menu the layout of page shifts when menu is open. This is similar to avatar menu fix we recently applied.

**Similar happens on organisation pages. I think this layout problem is present on every dot menu.**

### News
<img width="993" height="682" alt="Image" src="https://github.com/user-attachments/assets/6054affb-35e4-45d3-8232-f1b51234c585" />

### Organisation
<img width="1611" height="409" alt="Image" src="https://github.com/user-attachments/assets/70ede24f-a544-4e2e-b53c-cf54ba24fe6b" />

**Comment by ewan-escience** [comment]
You mean that the scroll bar disappears when you open such a menu? I don't see any other change or shift happening.

**Comment by dmijatovic** [comment]
> You mean that the scroll bar disappears when you open such a menu? I don't see any other change or shift happening.

In Chrome the layout shifts to the left and there is white space on the right side of the page. So there is difference in behaviour between browsers as in FF only the scrollbar dissappears which is desired/expected behaviour.

**This needs more investigation to determine the scope of the "bug" and optimal way to fix it.**

**Comment by ewan-escience** [comment]
> This needs more investigation to determine the scope of the "bug" and optimal way to fix it.

Didn't we have the same before on the login provider modal? I cannot find the issue/PR.

**Comment by dmijatovic** [comment]
No, we had this with user menu (top right). I fixed it as a part of a11y PR after I noticed this behavior. It seemed to me related to previous a11y fix I made to remove horizontal scroll problem at 400% zoom. Now I am not sure about this due to different behavior in Chrome and FF.

And I see on dev that it appears on login modal too in Chrome. It look like it appears all over now in all menus and modals in Chrome. I need to test it on more browsers and platforms to assess where unwanted behavior occures.

**Comment by dmijatovic** [comment]
I found the issue. 

**The layout problem seem to be caused by `Chrome plugin Handsfree for Web - Voice Control` I installed to test voice control. This plugin did not worked well for voice control for me anyway. It appears that this plugin injects `overflow:auto !important` forcing the page to always have overflow, while modals and menus insert overflow:hidden to prevent scrolling while modal is shown or menu is open.**

Because this issue triggered me to change the behavior of Feedback modal and User Menu I need to revert these changes :-(.
